src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -21,6 +21,16 @@ namespace Microsoft.PowerShell.Commands
     public sealed class RestartComputerCommand : CommandLineCmdletBase
     {
         // TODO: Support remote computers?
+        
+#region "Parameters"
+
+        /// <summary>
+        /// Force the operation to take place if possible.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Force { get; set; } = false;
```

Comment:
No need to init to default.
```suggestion
        public SwitchParameter Force { get; set; }
```

Reply:
I will remove it, but just an fyi I used Computer.cs as my style template and it is defaulted there.

Reply:
Implemented.

Reply:
Removing the default assignment `false` on `force` params may have caused validation tests to fail.  Re-added as a test.

Reply:
No, it doesn't affect the tests.